test/support/action/accountsLogin.spec.js

```diff
@@ -0,0 +1,30 @@
+import accountsLogin from "../../../src/support/action/accountsLogin.js";
+import { getNICEAccountsUrl } from "../../../src/support/utils";
```

Comment:
Looks like this import isn't being used?

Reply:
We have removed this line